### ISSUE TITLE
Improve Support for Gregtech 5u / GT New Horizons

### ISSUE THREAD

**Issue by jagoly** [John-Smith-Modded/JSTR-Modded-1.7.x#185]
I'm currently working on adding a bunch of guis for gregtech machines. I'll be using this issue to track things.

todo:

- Amplifabricator
- Autoclave
- BronzeAlloySmelter
- BronzeCompressor
- ChemicalReactor
- DistillationTower
- Electrolyzer
- FluidHeater
- FusionReactor
- Hammer
- LaserEngraver
- Packager
- PotionBrewer
- Press
- Replicator
- Sifter
- Slicer
- SteelAlloySmelter
- SteelCompressor
- SteelExtractor
- SteelHammer
- SteelMacerator
- Unpackager

done:

- Centrifuge
- Default (multiblocks)
- Distillery
- Fermenter

needs update:
- mixer (4 to 6 slots, done)

things I'm not sure about (need to check if used)
- NineXNine
- OilCracker
- everything outside of basicmachines

**Comment by jagoly** [comment]
I also added fluid input / output slots where appropriate

**Comment by jagoly** [comment]
I also removed the junk from the right, which has long since been used by the mod.

**Comment by TimDiekmann** [comment]
Any updates on this?

**Comment by jagoly** [comment]
not really, but the stuff I did get done is in my fork https://github.com/jagoly/JSTR-Modded-1.7.x

I might get back to this at some point, I'm not really able to play the game right now though

**Comment by TimDiekmann** [comment]
Take you time and thanks for responding!

**Comment by lesdmark** [comment]
If you would like to submit a pull request for the work you did I can take a look at it and go ahead and merge it with the master branch.
